source/examples/generated/flutter/app_services_test.snippet.access-app-by-id.dart

```diff
@@ -0,0 +1,23 @@
+// Create an App instance once on main isolate,
+// ideally as soon as the app starts
+final appConfig = AppConfiguration(APP_ID);
+final app = App(appConfig);
+final appId = app.id;
+// Later, access the App instance on background isolate
+await Isolate.spawn((List<Object> args) async {
+  final sendPort = args[0] as SendPort;
+  final appId = args[1] as String;
+
+  try {
+    final app = App.getById(appId); 
+
+    // ... Access App users 
+    final user = app?.currentUser!;
+
+    // ... Open and use the synced database as usual
```

Comment:
Should this be more general? Theoretically I suppose the isolate could be calling a Function, using GraphQL, or in the future maybe using MongoClient once that comes to Dart - should this comment be something like:

```suggestion
    // ... Use the App and user as needed.
```